packages/clerk-js/src/ui/components/KeylessPrompt/index.tsx

```diff
@@ -1,169 +1,396 @@
+                    opacity: 1;
+                  }
+                }
+
+                @keyframes text-shimmer {
+                  0% {
+                    background-position: 120% center;
+                  }
+                  30%,
+                  100% {
+                    background-position: -60% center;
+                  }
+                }
+
+                @keyframes text-shimmer-expanded {
+                  0% {
+                    background-position: 120% center;
+                  }
+                  30%,
+                  100% {
+                    background-position: -60% center;
+                  }
+                }
+              `}
+            >
+              Clerk is in keyless mode
+            </p>
+          </Flex>
+
+          {isExpanded && (
+            <button
+              onClick={() => setIsExpanded(false)}
+              aria-label='Close'
+              type='button'
+              css={css`
+                cursor: pointer;
+                margin-left: 0.75rem;
+                color: #8c8c8c;
+                transition: color 130ms ease-out;
+                :hover {
+                  color: #eeeeee;
+                }
+                animation: show-button 200ms cubic-bezier(0.4, 0, 0, 1.1) forwards;
+
+                @keyframes show-button {
+                  from {
+                    transform: scaleX(0.9);
+                    opacity: 0;
+                  }
+                  to {
+                    transform: scaleX(1);
+                    opacity: 1;
+                  }
+                }
+              `}
+            >
+              <svg
+                width='1rem'
+                height='1rem'
+                viewBox='0 0 16 16'
+                fill='none'
+                aria-hidden
+                xmlns='http://www.w3.org/2000/svg'
+              >
+                <path
+                  d='M3.75 8H12.25'
+                  stroke='currentColor'
+                  strokeWidth='1.5'
+                  strokeLinecap='round'
+                  strokeLinejoin='round'
+                />
+              </svg>
+            </button>
+          )}
         </Flex>
+
+        {isExpanded && (
+          <p
+            css={css`
+              color: #b4b4b4;
+              font-size: 0.75rem;
+              font-weight: 400;
+              line-height: 1rem;
+              max-width: 14.625rem;
+              animation: show-description 260ms ease-out forwards;
+
+              @keyframes show-description {
+                from {
+                  transform: translateY(-1.8px);
+                  opacity: 0;
+                }
+                to {
+                  transform: translateY(0);
+                  opacity: 1;
+                }
+              }
+            `}
+          >
+            We noticed your app was running without API Keys. Claim this instance by linking a Clerk account.{' '}
+            <Link
+              href='/'
```

Comment:
Ah, too late from me but I notice we need to update this link.

I think it will be `https://clerk.com/docs/keyless` but I'll confirm.